core/src/saros/communication/extensions/CancelResourceNegotiationExtension.java

```diff
@@ -21,8 +21,8 @@
  * <p>Furthermore you should "format" the XML output by adding XStream specific annotations to the
  * class fields.
  */
-@XStreamAlias(/* ProjectNegotiationCancel */ "PNCL")
-public class CancelProjectNegotiationExtension extends SarosSessionPacketExtension {
+@XStreamAlias(/* ResourceNegotiationCancel */ "RNCL")
```

Comment:
Breaks compatibility just because you renamed something. I think it is fine for now but the commit message just simply says it is a refactoring which is misleading.

Reply:
I got a bit liberal with the commit messages as there was a lot to do and I did not want to spend too much time writing the same stuff over and over.

And yes, this change theoretically breaks compatibility. But, with the general migration of the resource implementation to support reference points instead of "projects" (which this is a part of), we break compatibility anyways, so I did not think it really mattered to explicitly mention it.